libs/agno/agno/team/team.py

```diff
@@ -6753,11 +6785,16 @@ def save_session(self, session: TeamSession) -> None:
                 session.session_data["session_state"].pop("current_user_id", None)  # type: ignore
                 session.session_data["session_state"].pop("current_run_id", None)  # type: ignore
 
-            # scrub the member responses if not storing them
-            if not self.store_member_responses and session.runs is not None:
+            # scrub the member responses based on storage settings
+            if session.runs is not None:
                 for run in session.runs:
                     if hasattr(run, "member_responses"):
```

Comment:
Ok yes this deals with member_responses on the team response itself, but the member runs that are stored directly on the session aren't being dealt with. I.e. where you call `upsert_run` you have to deal with it